src/services/__tests__/email_service.test.ts

```diff
@@ -0,0 +1,27 @@
+import { emailService } from '../email_service';
+import { VisitorEmailTemplate } from '../templates';
+const emailOptions = {
+  to: 'hi@hello.com',
+  from: 'bye@seeya.com',
+  templateId: VisitorEmailTemplate.WELCOME_CB,
+  templateModel: {},
+};
+
+describe('Email Service', () => {
+  const email = emailService.init({ apiKey: 'POSTMARK_API_TEST' });
+  test('Config for single email', async() => {
```

Comment:
This test description doesn't really tell me what's being tested.

The test body looks like it's expecting an error. Why? What scenario is under test? What does the error code mean? Why is there no "success" case?

Reply:
FYI you can nest `describe` blocks for added context:
```ts
describe('Email service', () => {
  describe('::send', () => {
    test('Malformed options throws an exception', () => {
      // ...
    })

    // ...
  })

  describe('::sendBatch', () => {
    // ...
  })
})
```

Reply:
From what I can gather, postmark will always error with the test api key, it doesn't mock success results for you.